src/components/dashboard/SendLinkSummary.js

```diff
@@ -294,7 +295,7 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
             </BackButton>
           </Section.Row>
           <Section.Stack grow={3}>
-            <CustomButton onPress={handleConfirm} disabled={isCitizen === undefined} lodaing={loading}>
+            <CustomButton onPress={handleConfirm} disabled={isUndefined(isCitizen)} loading={loading}>
```

Comment:
the test for isCitizen should be removed. it is no longer relevant.
loading is not set to true when sending via link. in that case the double tap will again cause link to be created twice

Reply:
done in  https://github.com/GoodDollar/GoodDAPP/pull/1829/commits/03d00a4407493e8f7bf6639dd32ec7ce8ca06ac0